capuccino/capuccino_py.py

```diff
@@ -0,0 +1,28 @@
+from math import *
+from espresso.espresso_py import espresso
+
+class capuccino(espresso):
+    def __init__(self, c_coffee, c_sugar, c_water,  milk):
+        super().__init__(coffee = c_coffee, sugar = c_sugar, water = c_water)
+        self.milk = milk
+
+    def prepare_coffee(self):
+        super().prepare_coffee()
+        self.add_milk = espresso.cantitate_totala_cafea + self.milk
```

Comment:
If you use this variable only in this method you don't have to assign it to the object (you don't have to use self)

Reply:
amended